_api-reference/cat/cat-indices.md

```diff
@@ -15,24 +15,41 @@ redirect_from:
 The CAT indices operation lists information related to indexes, that is, how much disk space they are using, how many shards they have, their health status, and so on.
 
 
+<!-- spec_insert_start
+api: cat.indices
+component: endpoints
+-->
 ## Endpoints
-
 ```json
-GET _cat/indices/<index>
-GET _cat/indices
+GET /_cat/indices
+GET /_cat/indices/{index}
 ```
+<!-- spec_insert_end -->
 
-## Query parameters
 
-Parameter | Type | Description
-:--- | :--- | :---
-bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
-health | String | Limit indexes based on their health status. Supported values are `green`, `yellow`, and `red`.
-include_unloaded_segments | Boolean | Whether to include information from segments not loaded into memory. Default is `false`.
-cluster_manager_timeout | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
-pri | Boolean | Whether to return information only from the primary shards. Default is `false`.
-time | Time | Specify the units for time. For example, `5d` or `7h`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
-expand_wildcards | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
+<!-- spec_insert_start
+api: cat.indices
+component: query_parameters
+columns: Parameter,Type,Description,Default
+include_deprecated: false
+-->
+## Query parameters
+Parameter | Type | Description | Default
+:--- | :--- | :--- | :---
+`bytes` | String | The units used to display byte values. | 
+`cluster_manager_timeout` | String | The amount of time allowed to establish a connection to the cluster manager node. | 
+`expand_wildcards` | List or String | The type of index that wildcard patterns can match. | 
```

Comment:
Can we provide valid values for `expand_wildcards`?

Reply:
Updating the description in the `spec` now.